src/contexts/DatasetManagerContext.js

```diff
@@ -1,22 +1,50 @@
-// TEMPORARY
-// NOTE: incoporates the refinbio API client to manage the dataset
 import { createContext, useMemo } from 'react'
-import { useLocalStorage } from 'hooks/useLocalStorage'
+import { useRefinebio } from 'hooks/useRefinebio'
 
 export const DatasetManagerContext = createContext({})
 
 export const DatasetManagerContextProvider = ({ children }) => {
-  const [dataset, setDataset] = useLocalStorage('dataset', null) // TEMPORARY
-  const [datasetId, setDatasetId] = useLocalStorage('datasetId', null)
+  const {
+    dataset: datasetState,
+    setDataset: setDatasetState,
+    datasetId: datasetIdState,
+    downloadOptions: downloadOptionsState,
+    setDownloadOptions: setDownloadOptionsState,
+    setDatasetId: setDatasetIdState,
+    email: emailState,
+    token: tokenState
+  } = useRefinebio()
```

Comment:
I think a better pattern would be to combine all of the global contexts (ex. DatasetManagerContext) into `RefinebioContext` instead of pulling everything from the main context. The benefit of inverting these would mean that `RefinebioContext.Provider` would be the only provider in our  in our `_app.js` and we would still be able to use the hooks as is because they would be scoped correctly.

Reply:
The reason I did it this way rather than removing `DatasetManagerContext` is that I wanted to have some flexibility in case the state might only needed to be defined at the `DatasetManagerContext` level, considering it's still in the development stage. Can we keep it for the time being and delete it later as necessary?

Reply:
I will create an issue for this.